src/Fixer/Whitespace/BlankLineBeforeStatementFixer.php

```diff
@@ -343,7 +399,7 @@ private function insertBlankLine(Tokens $tokens, int $index): void
                 $tokens[$prevIndex] = new Token([T_WHITESPACE, $lineEnding.$prevToken->getContent()]);
             }
         } else {
-            $tokens->insertAt($index, new Token([T_WHITESPACE, $lineEnding.$lineEnding]));
+            $tokens->insertSlices([$index => new Token([T_WHITESPACE, $lineEnding.$lineEnding])]);
```

Comment:
why the change?

Reply:
It asks me to do this 
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/dca212adc0f12b081eb7566af0025f9ab1665c7b/tests/Test/AbstractFixerTestCase.php#L305

Reply:
it's not to blindly replace method "as is".
it's to instead of `foreach (array) => insert(singleItem)` do `foreach (array) => build insertionArray; insertSlices(insertionArray)`, so you call `insertSlices` ONCE for whole file

Reply:
Ouu. Thanks. If PR, in general, will be accepted, I will refactor it in the correct way.